src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -144,7 +158,27 @@ def create(input_dataset, target, feature=None,
     classes = sorted(classes)
     class_to_index = {name: index for index, name in enumerate(classes)}
 
-    def update_progress(cur_loss, iteration):
+    if isinstance(validation_set, _tc.SFrame):
+        _raise_error_if_not_drawing_classifier_input_sframe(
+            validation_set, feature, target)
+        is_validation_stroke_input = (validation_set[feature].dtype != _tc.Image)
```

Comment:
It doesn't look like `is_validation_stroke_input` is getting used.

Reply:
Thanks for the catch, the next line should use it.